src/Framework/MSBuildEventSource.cs

```diff
@@ -79,11 +79,12 @@ public void BuildStop()
         /// <summary>
         /// Call this method to notify listeners of information of how a project file built.
         /// <param name="projectPath">Filename of the project being built.</param>
+        /// <param name="targets">Names of the targets that built.</param>
         /// </summary>
-        [Event(5, Keywords = Keywords.All | Keywords.PerformanceLog)]
-        public void BuildProjectStart(string projectPath)
+        [Event(5, Keywords = Keywords.All | Keywords.PerformanceLog, Version = 1)]
```

Comment:
are there any compat consequences of this change? @YuliiaKovalova

Reply:
@JanProvaznik I add the Version =1 since this comment https://github.com/JaynieBai/msbuild/blob/0c1be9946f2a10ff2d93d5e486710c7139cf01a3/src/Framework/MSBuildEventSource.cs#L12